package.json

```diff
@@ -60,17 +60,11 @@
     "pinkie-promise": "^2.0.0"
   },
   "devDependencies": {
+    "ava": "*",
```

Comment:
I'm not sure I understand. If you depend on a version range such as `^1.2.3`, then you can still test all releases that are supposedly backwards-compatible with the version that you depend on (e.g. `1.4.6`, `1.20.99`).
When you do release a version `2.0.0`, that means there are intentional breaking changes and users are expected to update existing code before upgrading the dependency.

As far as I can see, `*` just completely misses the point of semver for no good reason.